### PR TITLE
STREAM-120 Improve oracle/snowflake schema crawler performance

### PR DESCRIPTION
**Pull request by akuma432**
* added snowflake jdbc dependency
* implemented logic to add tables filter in the oracle/snowflake custom queries.
* STREAM-125 - Add hive and impala jdbc drivers.
* STREAM-121 - Support attribute "errorIntegration" under destination in ingest-config
* STREAM-122 - Document "userDefinedTable" and "genericProperties" feature.

### CODE REVIEW

File: customer-docs/user-manual.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ Streamliner uses configuration files which applied to templates to create DDL an
 2. `scripts` DDL and DML creation - Applies configuration properties to templates to produce DDL and DML scripts.  Templates are a collection of operations producing a data ingestion pipeline for a single table.
 
 ## Configuration
+Streamliner 5.x is compatible with Java 11.
```
Comment (satish-sanjeev) [inline customer-docs/user-manual.md:73]: Move compatible java version to a separate heading.

**Review by satish-sanjeev — APPROVED** [review]
Approved when comments are resolved

File: customer-docs/user-manual.md [hunk 2]
```diff
@@ -84,6 +86,7 @@ The base level pipeline configuration has 3 properties that define the pipeline.
 | pipeline | String | True | The name of the pipeline template (ex: snowflake-snowpipe-append or truncate-reload) |
 | source | Source | [Data Source Configuration](#data-source-configuration) |
 | destination | Destination | [Destination Configuration](#destination-configuration) |
+| genericProperties | Map[String, String] | False | It can be used to pass any generic parameter that are not pre-defined. |
```
[the remark this answers is not in the document]
Comment (akuma432) [reply customer-docs/user-manual.md:108]: updated

File: customer-docs/user-manual.md (hunk 1 above)
Comment (akuma432) [reply customer-docs/user-manual.md]: updated

**Pull request merged**